feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go

```diff
@@ -163,9 +173,22 @@ func TestPrefixSetWithCLIConfig(t *testing.T) {
 	v4PrefixSet.GetOrCreatePrefix("173.48.128.0/20", mskLen)
 	v4PrefixSet.GetOrCreatePrefix("173.45.128.0/20", mskLen)
 
-	gnmi.Replace(t, dut, gnmi.OC().RoutingPolicy().DefinedSets().PrefixSet("TAG_3_IPV4").Config(), v4PrefixSet)
-	prefixSet := gnmi.Get[*oc.RoutingPolicy_DefinedSets_PrefixSet](t, dut, gnmi.OC().RoutingPolicy().DefinedSets().PrefixSet("TAG_3_IPV4").State())
+	gnmi.Replace(t, dut, gnmi.OC().RoutingPolicy().DefinedSets().PrefixSet(tag3IPv4).Config(), v4PrefixSet)
+	prefixSet = gnmi.Get[*oc.RoutingPolicy_DefinedSets_PrefixSet](t, dut, gnmi.OC().RoutingPolicy().DefinedSets().PrefixSet(tag3IPv4).State())
 	if len(prefixSet.Prefix) != 22 {
 		t.Errorf("Prefix set has %v prefixes, want 22", len(prefixSet.Prefix))
 	}
 }
+
+// findProcessByName uses telemetry to find out the PID of a process
+func findProcessByName(ctx context.Context, t *testing.T, dut *ondatra.DUTDevice, pName string) uint32 {
```

Comment:
move to some suitable place in /internal

Perhaps /internal/system (modeling after the OC top level tree where process names are found `/system/processes/process/state/name`)